src/microbe_stage/MicrobeStage.tscn

```diff
@@ -351,7 +351,7 @@ MenuPath = NodePath("../PauseMenu")
 AtpLabelPath = NodePath("BottomRight/DataValue/Margin/VBox/ATPValue")
 HpLabelPath = NodePath("BottomRight/DataValue/Margin/VBox/HPValue")
 PopulationLabelPath = NodePath("BottomRight/VBoxContainer/PopulationData/Value")
-PatchOverlayPath = NodePath("PatchNameOverlay")
+PatchOverlayPath = NodePath("../PatchNameOverlay")
```

Comment:
I foresee this causing a bit of non-optimal design in the future (https://github.com/Revolutionary-Games/Thrive/issues/3741).